spec/rubocop/cop/layout/empty_lines_around_arguments_spec.rb

```diff
@@ -38,18 +38,6 @@
         .to eq(['Empty line detected around arguments.'])
     end
 
-    it 'registers offense for empty line in-between one arg' do
-      inspect_source(<<-RUBY.strip_indent)
-        do_stuff([
-          "some text",
-
-          "some more text"
-        ])
-      RUBY
-      expect(cop.messages)
-        .to eq(['Empty line detected around arguments.'])
-    end
```

Comment:
This is precisely the behavior that is getting removed.  `do_stuff` has one arg that is an array. There are no empty lines before or after the arg.  This cop no longer cares what happens inside the arg itself.